src/UIKit/UIEventButtonMaskExtensions.cs

```diff
@@ -0,0 +1,29 @@
+//
+// UIEventButtonMaskExtensions.cs
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+#if IOS
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+namespace UIKit {
+	[Introduced (PlatformName.iOS, 13,4, PlatformArchitecture.All)]
+	[BindingImpl (BindingImplOptions.Optimizable)]
+	public static partial class UIEventButtonMaskExtensions {
+
+        [DllImport (Constants.UIKitLibrary)]
+        static extern nint UIEventButtonMaskForButtonNumber (nint buttonNumber);
+
+        public static UIEventButtonMask Convert (nint buttonNumber)
+		{
+            return (UIEventButtonMask) (long) UIEventButtonMaskForButtonNumber (buttonNumber);
+		}
```

Comment:
Good lord... (*&^%$# formatting why can't VSMAC do the right thing.... ^%$^#$%^&$#